scanner/src/main/kotlin/scanners/ScanCodeResultParser.kt

```diff
@@ -138,7 +138,20 @@ private fun getLicenseId(license: JsonNode): String {
     // in ScanCode, and it should have always been using null instead.
     var name = license["spdx_license_key"].textValueOrEmpty()
 
-    if (name.isEmpty()) {
+    // There is only one [1] license in the latest ScanCode 3.2.x which has a "LicenseRef-*" identifier set as
```

Comment:
Nit: Shouldn't the `[1]` reference be put *after* the word "license"?

Reply:
Done.